### PR TITLE
Added Filter Issues to What's New v100

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100?branch=pr-en-us-1869#filter-messages-in-the-issues-tool-by-severity-and-browser-target

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md [hunk 1]
```diff
@@ -111,6 +111,35 @@ Now in Microsoft Edge 100, the selected theme is applied to the Waterfall view o
 For more information about using themes from Visual Studio Code in DevTools, see [Apply a color theme to DevTools](../../../customize/theme.md).  For more information about the Waterfall view for requests in the Network tool, see [Network features reference](../../../network/reference.md#display-the-timing-relationship-of-requests).
 
 
+<!-- ====================================================================== -->
+## Filter messages in the Issues tool by severity and browser target
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md:115]: Can we move this section to after "The Network tool now displays how a request was fulfilled"? I believe that as long as we don't change the actual section titles, our in-product links will still work

**Review by zoherghadyali — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md [hunk 2]
```diff
@@ -73,6 +73,35 @@ To learn more about the request log in the Network tool, see [Filter requests by
...
+
+The **Browser** filter only applies to browser compatibility feedback, which is shown in the **Compatibility** section of the **Issues** list.  By default, the **Browser** filter shows messages for the top browsers, as defined by [Browserslist](https://github.com/browserslist/browserslist#queries):
+
+![The Browsers drop-down list, with "Top browsers" selected.](devtools-100-images/browser-filters-not-filtered.png)
+
+In the following example, messages about Firefox and Firefox for Android are filtered out:
+
+![Messages about Firefox browser filtered out (removed).](devtools-100-images/browser-filters-filtered-out.png)
+
+For more information, see [Filter issues](../../../issues/index.md#filter-issues) in _Find and fix problems using the Issues tool_.
```
Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md:102]: FYI, this anchor link did not scroll down to the relevant heading.

**Review by hxlnt — APPROVED** [review]
One small comment but approved. Thanks!

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md:102]: It works on my machine; very high likelihood it's a temp caching issue involving review.dmc temp branch server.

**Pull request merged**